Makefile

```diff
@@ -289,6 +289,8 @@ release/new:
 .PHONY: release/gen-templates
 release/gen-templates:
 	bazel run //hack/crdbversions:crdbversions -- -operator-version $(APP_VERSION) -crdb-versions $(PWD)/crdb-versions.yaml -repo-root $(PWD)
+	bazel run //config/crd:manifest.preview > install/crds.yaml
```

Comment:
Please add comments what we are doing

Reply:
Added some more details to the make target comment, as well as on the kustomization macros in the build files